.travis.yml

```diff
@@ -13,13 +13,10 @@ addons:
       - google-chrome-stable
       - g++-4.8
   firefox: 'latest-esr'
-cache:
-  directories:
-  - node_modules
 env:
   matrix:
-    - EMBER_TRY_SCENARIO=ember-2-8
-    - EMBER_TRY_SCENARIO=default
+    - EMBER_TRY_SCENARIO=ember-2-11
```

Comment:
As we've discussed, in the previous version, Bunsen actually never gets run with `ember: 2.8` (With `ember-source` not being set as null in config/ember-try, CI will always run against ember-source presented in package.json). I tried to run it against `ember: 2.8.0`, but I'm seeing extra test failure. Since it never runs against 2.8, I won't bother to fix these tests at this moment.